src/DbUtils.php

```diff
@@ -536,7 +536,7 @@ public function countElementsInTable($table, $condition = [])
      *
      * @param string|array   $table     table name(s)
      * @param string         $field     field name
-     * @param ?string|?array $condition array of criteria
+     * @param array          $condition array of criteria
```

Comment:
Same here, there is a !is_array() check so I guess condition can be anything

Reply:
We're letting something else through, just to say it's deprecated, so just `array` in the comment seems appropriate. no?

Reply:
I guess you can see it like that, but then phpstan will take its revenge on higher levels: https://phpstan.org/r/956dd902-4675-4bf2-bb44-8d0fe2a21f0d

Reply:
phpodc properties annotations does not take care of depreciations; they mustdescribe what can be passed to the method. Here, we really expect a string or an array from a code point of view, bih are correct, depreciation are handled later. Once deprecated code will be removed, phpdoc will be updated.